UNRELEASED.md

```diff
@@ -1,3 +1,11 @@
 **Changed:**
 
 `bpk-component-button`: Cleaned up the onClick functionality to remove a wrapper that is no longer necessary
+
+**Fixed:**
```

Comment:
Based on the change and we are removing props - therefore the contract I would say can you mark this as breaking? Thanks

Reply:
I think this change doesn't affect anyone as the bit which was removed didn't have any effect as far as we can see 🤔. Also the props which were removed didn't have any effect either, but given consumers still pass them (as they are marked as required), might be worth flagging so that they can be removed.

Reply:
The change breaks the API (removing properties), but given that we don't have any typechecks in the released library, it will be backwards compatible with existing consumers. If they pass these two props in, then they will simply be ignored. I have marked this as breaking anyway, for clarity. Thanks both! 😄